kimchi/src/proof.rs

```diff
@@ -36,6 +36,8 @@ pub struct ProofEvaluations<Field> {
     pub generic_selector: Field,
     /// evaluation of the poseidon selector polynomial
     pub poseidon_selector: Field,
+    /// evaluation of the cairo selector polynomial
+    pub cairo_selector: [Field; crate::circuits::polynomials::turshi::CIRCUIT_GATE_COUNT],
```

Comment:
import turshi so you can write `turshi::CIRCUIT_GATE_COUNT`

Reply:
like lookup this should be an `Option`

Reply:
I see, like above with the `cairo` and `cairo8`